src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```diff
@@ -170,23 +170,12 @@ class TaskDeploymentService {
         return pairs.map { KeyValuePair.builder().name(it.first).value(it.second).build() }
     }
 
-    /**
-     * Helper method to initialise the lateinit vars [taskAssumeRoleString] and [taskRolePolicyString] by
-     * converting the associated `@Value` parameters to Strings and replacing `ADDITIONAL_PERMISSIONS` in
-     * [taskRolePolicyString] with the provided [additionalPermissions]
-     *
-     * @return [Pair] of [taskRolePolicyString] to [taskAssumeRoleString] for ease of access.
-     */
-    fun parsePolicyDocuments(additionalPermissions: List<String>): Pair<String, String> {
-        var replaceString = ""
-        if (additionalPermissions.isNotEmpty()) {
-            logger.info("Adding permissions to containers", "permissions" to additionalPermissions.joinToString())
-            val permissionsJson = additionalPermissions.joinToString(prefix = "\"", separator = "\",\"", postfix = "\"")
-            replaceString = "$permissionsJson,"
-        }
-        taskAssumeRoleString = taskAssumeRoleDocument.inputStream.bufferedReader().use { it.readText() }
-        taskRolePolicyString = taskRolePolicyDocument.inputStream.bufferedReader().use { it.readText() }
-                .replace("ADDITIONAL_PERMISSIONS", replaceString)
-        return taskRolePolicyString to taskAssumeRoleString
+    fun parseMap (cognitoGroups: List<String>, userName: String): Map<String, List<String>> {
```

Comment:
Since this is a non-specific method name, some documentation or indication of what it does would be useful